docs/advanced-stream.md

```diff
@@ -1,51 +1,171 @@
+        "youTubeId": "lS7AE0PJENo",
+        "imageUrl": "/static/advanced-stream/lS7AE0PJENo.jpg"
+    },
+    {
+        "name": "Sprite Data",
+        "description": "Join Richard, Daryl, Shannon, and Joey as they make some friends! In this stream we talk about how to add custom properties (like \"name\", or \"lives\") to a sprite using the arcade-sprite-data extension",
+        "youTubeId": "oIdu_k8d-94",
+        "imageUrl": "/static/advanced-stream/oIdu_k8d-94.jpg"
+    },
+    {
+        "name": "Using settings Pt. 2",
+        "description": "Join Richard, Daryl, Shannon, and Joey as they continue to make a pirate game! In this stream we talk about more settings like strings and arrays and also how to reset your settings when you win or lose the game",
+        "youTubeId": "ogpAcdx2pZY",
+        "imageUrl": "/static/advanced-stream/ogpAcdx2pZY.jpg"
+    },
+    {
+        "name": "Using settings",
+        "description": "Join Richard, Daryl, Shannon, and Joey as they make a pirate game! This stream is a great one to watch if you want to learn how to use settings to save a player's progress",
+        "youTubeId": "qafMt6bPY58",
+        "imageUrl": "/static/advanced-stream/qafMt6bPY58.jpg"
+    },
+    {
+        "name": "Boots in Space",
+        "description": "Join Joey, Richard, Daryl, and Shannon as we make a game about cleaning up boots",
+        "youTubeId": "8cjyVY6yfGM",
+        "imageUrl": "/static/advanced-stream/8cjyVY6yfGM.jpg"
+    },
+    {
+        "name": "Climbing a Tree",
+        "description": "Join Richard, Shannon, Daryl, and Joey as they create a game loosely based on doodle jump! There's an adorable squirrel in this one too",
+        "youTubeId": "M4n2j27jAZI",
+        "imageUrl": "/static/advanced-stream/M4n2j27jAZI.jpg"
+    },
+    {
+        "name": "Survival Body Building",
+        "description": "Join Richard, Shannon, Daryl, and Joey as they continue working on their survival game",
+        "youTubeId": "glQRlOyNgtQ",
+        "imageUrl": "/static/advanced-stream/glQRlOyNgtQ.jpg"
+    },
+    {
+        "name": "Mood lighting",
+        "description": "Join Richard, Shannon, Daryl, and Joey as they continue working on their survival game",
+        "youTubeId": "w7Y6Do9HPU4",
+        "imageUrl": "/static/advanced-stream/w7Y6Do9HPU4.jpg"
+    },
+    {
+        "name": "Making a survival game",
+        "description": "Join Richard, Shannon, Daryl, and Joey as they create another randomly generated game idea! Today we started making a survival game and show how to make a game HUD",
+        "youTubeId": "SvXeJevUkb4",
+        "imageUrl": "/static/advanced-stream/SvXeJevUkb4.jpg"
+    },
+    {
+        "name": "A Colorful Tundra Monkey",
+        "description": "Join Joey, Daryl, Richard, and Shannon as we make game about throwing fruit at animals in the tundra!\n\nThe code from today is here: https://makecode",
```

Comment:
this one too

Reply:
right i should only be taking the first paragraph.